pkg/admission/chain.go

```diff
@@ -16,22 +16,37 @@ limitations under the License.
 
 package admission
 
-import clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+import (
+	"time"
+
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/controller/framework/informers"
+)
 
 // chainAdmissionHandler is an instance of admission.Interface that performs admission control using a chain of admission handlers
 type chainAdmissionHandler []Interface
 
 // NewFromPlugins returns an admission.Interface that will enforce admission control decisions of all
 // the given plugins.
-func NewFromPlugins(client clientset.Interface, pluginNames []string, configFilePath string) Interface {
+func NewFromPlugins(client clientset.Interface, pluginNames []string, configFilePath string) (Interface, error) {
 	plugins := []Interface{}
 	for _, pluginName := range pluginNames {
 		plugin := InitPlugin(pluginName, client, configFilePath)
 		if plugin != nil {
 			plugins = append(plugins, plugin)
 		}
 	}
-	return chainAdmissionHandler(plugins)
+
+	informer := informers.CreateSharedNamespaceInformer(client, 5*time.Minute)
```

Comment:
I'd recommend moving this to cmd/kube-apiserver/app/server.go, where `admissionController` is set up. Unless @deads2k has a better place?